backend/src/main/java/de/echochamber/backend/repo/UserRepository.java

```diff
@@ -0,0 +1,21 @@
+package de.echochamber.backend.repo;
+
+import de.echochamber.backend.model.UserEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface UserRepository extends JpaRepository<UserEntity, Long> {
+
+    Optional<UserEntity> findByFirstName(String firstName);
+
+    Optional<UserEntity> findByFirstNameContains(String firstName);
```

Comment:
firstbyname macht glaube ich keinen sind wenn du mehre markus in der db hast findest du immer nur den ersten